modules/effects/src/main/scala/dev/profunktor/redis4cats/redis.scala

```diff
@@ -1173,6 +1176,30 @@ private[redis4cats] class BaseRedis[F[_]: Concurrent: ContextShift, K, V](
       .futureLift
       .map(x => Option(Double.unbox(x)))
 
+  override def zPopMin(key: K, count: Long): F[List[ScoreWithValue[V]]] =
+    async
+      .flatMap(c => F.delay(c.zpopmin(key, count)))
+      .futureLift
+      .map(_.asScala.toList.map(_.asScoreWithValues))
+
+  override def zPopMax(key: K, count: Long): F[List[ScoreWithValue[V]]] =
+    async
+      .flatMap(c => F.delay(c.zpopmax(key, count)))
+      .futureLift
+      .map(_.asScala.toList.map(_.asScoreWithValues))
+
+  override def bzPopMin(timeout: Duration, keys: NonEmptyList[K]): F[Option[(K, ScoreWithValue[V])]] =
```

Comment:
Same questions here: if these commands are blocking until there's a result maybe the `Option` is redundant?